src/main/java/org/reactome/release/qa/check/InferredEvidenceTypeCheck.java

```diff
@@ -0,0 +1,59 @@
+package org.reactome.release.qa.check;
+
+import org.gk.model.GKInstance;
+import org.gk.model.ReactomeJavaConstants;
+import org.reactome.release.qa.annotations.SliceQACheck;
+import org.reactome.release.qa.common.AbstractQACheck;
+import org.reactome.release.qa.common.QACheckerHelper;
+import org.reactome.release.qa.common.QAReport;
+
+import java.util.Collection;
+
+/**
+ *  This QA check was mostly used during the CoV-1-to-CoV-2 inference process. For now it is being kept, but may
+ *  be removed, given it was for a specific curations (August 2020).
+ *
+ *  Since CoV-1-to-CoV-2 inference involved using a modified version of orthoinference, this check was created to find
+ *  instances that had a 'Inferred from Electronic Annotation' EvidenceType, so Curators could prioritize their work.
+ *
+ *  @author jcook
+ */
+@SliceQACheck
+public class InferredEvidenceTypeCheck extends AbstractQACheck {
+
+    @Override
+    public QAReport executeQACheck() throws Exception {
+        QAReport report = new QAReport();
+
+        Collection<GKInstance> events = dba.fetchInstancesByClass(ReactomeJavaConstants.Event);
+        for (GKInstance event : events) {
+            // Orthoinference adds a typical EvidenceType instance to projected instances that have
+            // a displayName of 'Inferred from Electronic Annotation'.
+            GKInstance evidenceType = (GKInstance) event.getAttributeValue(ReactomeJavaConstants.evidenceType);
+            if (evidenceType != null && evidenceType.getDisplayName().contains("Inferred from Electronic Annotation")) {
+                report.addLine(getReportLine(event, evidenceType));
+            }
+
+        }
```

Comment:
Since thie checks all events for an evidence type, are Cov-2 events the only ones that are reported because there are no other electronically inferred instances in the database?  If so, is it okay to have it stay as checking all events or would it be good to filter based on having a Cov-2 species?

Reply:
Yes, everything described above is correct/the plan. Even though this QA test was created in response to the CoV-1-to-Cov-2 work, its plausible in the future that the orthoinference process will be used in a similar way to generate inference databases, and so this type of check would be a basic QA requirement. In the future, the only thing that might need to change is the commenting referencing the CoV-1-to-CoV-2 work, but for now I'd like it kept so we can find those tests used during the project